tools/generate-codeowners.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Should we add some CI automation that would run this script on PRs and check the diff, so that no unintentional manual changes make it to `CODEOWNERS`. And that added/removed teams are always covered in `CODEOWNERS`.

Reply:
Ah yes, this is what I had in mind but the basic workflow is not yet in place. I have the base for the workflow in #262, so depending which PR goes in first I can add this check on top.